python/pyspark/sql/functions.py

```diff
@@ -962,9 +993,9 @@ def add_months(start, months):
     """
     Returns the date that is `months` months after `start`
 
-    >>> df = spark.createDataFrame([('2015-04-08',)], ['d'])
-    >>> df.select(add_months(df.d, 1).alias('d')).collect()
-    [Row(d=datetime.date(2015, 5, 8))]
+    >>> df = spark.createDataFrame([('2015-04-08',)], ['dt'])
```

Comment:
I think changing arbitrary name to `dt` or `ts` is fine but not worth changing `t`, `timestamp`, `d` or `date`. These are already readable.